firedrake/mg/mesh.py

```diff
@@ -89,6 +96,9 @@ def MeshHierarchy(mesh, refinement_levels,
     :arg refinement_levels: the number of levels of refinement
     :arg refinements_per_level: the number of refinements for each
         level in the hierarchy.
+    :arg degree: the degree of the finite element space used for
```

Comment:
In line with our policy, please use modify this docstring to be in [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) format.